v2/schema/example.jsonld

```diff
@@ -0,0 +1,89 @@
+{
+  "@context": "https://raw.githubusercontent.com/geoblacklight/geoblacklight-schema/json-ld-schema/v2/schema/context.jsonld",
+  "@id": "http://hdl.handle.net/2451/34502",
+  "@type": "dcat:Dataset",
+  "accessLevel": "public",
+  "creator": [
+    "GIS Lab, Newman Library, Baruch CUNY"
+  ],
+  "description": "The subway complexes layer was created to represent ridership data for the NYC subway system (Metropolitan Transportation Authority, or MTA).",
+  "distribution": [
+    {
+      "@type": "dcat:Distribution",
+      "conformsTo": "http://www.opengis.net/def/serviceType/ogc/wms",
+      "accessURL": "https://maps-public.geo.nyu.edu/geoserver/sdr/wms",
+      "layerId": "sdr:nyu_2451_34502",
+      "describedBy": "https://archive.nyu.edu/retrieve/74755/nyu_2451_34502_doc.zip",
+      "describedByType": "application/pdf",
+      "description": "2015 New York City Subway Complexes and Ridership WMS Service"
+    },
+    {
+      "@type": "dcat:Distribution",
+      "conformsTo": "http://www.opengis.net/def/serviceType/ogc/wfs",
+      "accessURL": "https://maps-public.geo.nyu.edu/geoserver/sdr/wfs",
+      "layerId": "sdr:nyu_2451_34502",
+      "describedBy": "https://archive.nyu.edu/retrieve/74755/nyu_2451_34502_doc.zip",
+      "describedByType": "application/pdf",
+      "description": "2015 New York City Subway Complexes and Ridership WFS Service"
+    },
+    {
+      "@type": "dcat:Distribution",
+      "conformsTo": "http://www.esri.com/library/whitepapers/pdfs/shapefile.pdf",
```

Comment:
Is this what you were referring to? https://github.com/geoblacklight/geoblacklight/blob/master/lib/geoblacklight/constants.rb#L12